homeassistant/components/go2rtc/__init__.py

```diff
@@ -156,8 +156,9 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         _LOGGER.warning("Could not connect to go2rtc instance on %s (%s)", url, err)
         return False
     except Go2RtcVersionError as err:
-        _LOGGER.warning("The go2rtc server version is not supported, %s", err)
-        return False
+        raise ConfigEntryNotReady(
+            f"The go2rtc server version is not supported, {err}"
```

Comment:
Will the error explain what version is required?

Reply:
Yes, it's explained. It's not very visible though since it's logged at debug level from `homeassistant.config_entries`:
```
2024-11-05 11:06:27.513 DEBUG (MainThread) [homeassistant.config_entries] Config entry 'go2rtc' for go2rtc integration not ready yet: The go2rtc server version is not supported, server version '1.9.4' not >= 1.9.5 and < 2.0.0; Retrying in 5 seconds
```